src/search.js

```diff
@@ -511,6 +514,11 @@ export class SearchElement extends LitElement {
         window.location.href = selected.href;
       }
     }
+
+    if (e.keyCode === 27) {
```

Comment:
```suggestion
    # If "Esc" key is pressed.
    if (e.keyCode === 27) {
```

Reply:
:snake:

Reply:
I think you can also do (https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code, https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key)

```suggestion
    if (e.key === "Escape") {
```

keycode is deprecated.

Reply:
Oh, I did not know that. Should I change other instances of `.keyCode` in this function as well? cc @humitos

Reply:
I didn't know either. Sounds good to me 👍